sphinx_gallery/_static/gallery.css

```diff
@@ -143,8 +150,13 @@ ul.sphx-glr-horizontal img {
   height: auto !important;
 }
 
+p.sphx-glr-signature {
+  text-align: right;
+  margin-left: auto;
+  display: table;
+}
+
```

Comment:
You can put 
margin-left: auto;
display: table;

in the next class and you get the same effect without having this separate class